csp-tech-radar.csv

```diff
@@ -26,11 +26,11 @@ OpenJDK11,adopt,platforms,TRUE,"When using any of the languages that run on the
 Windows Server,hold,platforms,TRUE,The Core Platform is our department's deployment infrastructure of choice. Avoid anything that requires Windows Server operating system to run.
 OracleJDK8,hold,platforms,TRUE,"Oracle's JDK8 reached 'End of Public Updates' in January 2019. Any new public updates will only be released for <strong>personal use only</strong> and <strong>should not be used</strong> by Sky in a commercial capacity! To avoid not being able to recieve security updates, move off OracleJDK8 and start using OpenJDK alternatives. Ideally OpenJDK11. Follow this <a href='https://developer.bskyb.com/wiki/display/CSEH/OpenJDK+11+Migration+Guide'>JDK11 Migration Guide</a> for services on the Core Platform."
-React JS,adopt,languages & frameworks,TRUE,"In the avalanche of front-end JavaScript frameworks, React.js stands out due to its design around a reactive data flow. Allowing only one-way data binding greatly simplifies the rendering logic and avoids many of the issues that commonly plague applications written with other frameworks. This library has been widely adopted across Sky’s SWE department as the User Interface library of choice and there is now a large wealth of knowledge around it in the department."
+React JS + TypeScript, trial, languages & frameworks,TRUE,"React.js stands out due to its design around a reactive data flow. Allowing only one-way data binding which greatly simplifies the rendering logic and avoids many of the issues that commonly plague applications written with other frameworks. This library has been widely adopted across Sky and there is a wealth of knowledge across the department. React components, however, with vanilla JavaScript have to rely on linters and prop-types to ensure code quality and catch bugs. TypeScript simplifies JavaScript code, making it easier to read and debug and provides a richer environment for spotting common errors as you type the code. It also follows OOP concepts, and has a powerful type system including generics. There is a community created flavor of create-react-app package that supports TypeScript. It’s called create-react-app-typescript and it can get you up and running easily if you don’t want a custom project setup."
```

Comment:
Should this not be `adopt` instead of `trial`.
```suggestion
React JS + TypeScript,adopt, languages & frameworks,TRUE,"React.js stands out due to its design around a reactive data flow. Allowing only one-way data binding which greatly simplifies the rendering logic and avoids many of the issues that commonly plague applications written with other frameworks. This library has been widely adopted across Sky and there is a wealth of knowledge across the department. React components, however, with vanilla JavaScript have to rely on linters and prop-types to ensure code quality and catch bugs. TypeScript simplifies JavaScript code, making it easier to read and debug and provides a richer environment for spotting common errors as you type the code. It also follows OOP concepts, and has a powerful type system including generics. There is a community created flavor of create-react-app package that supports TypeScript. It’s called create-react-app-typescript and it can get you up and running easily if you don’t want a custom project setup."
```